src/TriangleNodesProvisioner.cpp

```diff
@@ -61,7 +61,29 @@ namespace blitzdg {
         BCmap{ new index_hashmap()},
         Mesh2D { _MeshManager },
         Nodes1D{ new Nodes1DProvisioner(_NOrder, 5, -1.0, 1.0) },
-		Jacobi{}, Vandermonde{}, LinSolver{}, Inverter{}
+		Jacobi{}, Vandermonde{}, LinSolver{}, Inverter{}, DGContext{
+            Np: NumLocalPoints,
+            Nfp: NumFacePoints,
+            K: NumElements,
+            NumFaces: NumFaces,
+            Filt: *Filter,
+            x: *xGrid,
+            y: *yGrid,
+            Fscale: *Fscale,
+            J: *J,
+            rx: *rx,
+            sx: *sx,
+            ry: *ry,
+            sy: *sy,
+            nx: *nx,
+            ny: *ny,
+            Dr: *Dr,
+            Ds: *Ds,
+            bcHash: *BCmap,
+            vmapM: *vmapM,
+            vmapP: *vmapP,
+            Lift: *Lift
+        }
```

Comment:
This method of initialization is not part of C++. I believe its another gcc extension.

Reply:
Ah, ok. Fixed.